numpy/lib/nanfunctions.py

```diff
@@ -169,7 +169,7 @@ def _remove_nan_1d(arr1d, overwrite_input=False):
     s = np.nonzero(c)[0]
     if s.size == arr1d.size:
         warnings.warn("All-NaN slice encountered", RuntimeWarning,
-                      stacklevel=5)
+                      stacklevel=6)
```

Comment:
I triple checked this and it seemed just way too low before.  It can even now sometimes point inside (but that is due to use at different depth).  (Same below.)